src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -254,10 +254,18 @@ export const exchangeStitchingEnvironment = ({
       orders(first: Int, last: Int, after: String, before: String, mode: CommerceOrderModeEnum, sellerId: String, sort: CommerceOrderConnectionSortEnum, states: [CommerceOrderStateEnum!]): CommerceOrderConnectionWithTotalCount
     }
 
+    extend input CommerceSubmitOrderWithOfferInput {
+      note: String
+      artworkId: ID
+    }
```

Comment:
Since `submitOrderOffer` is a separate mutation from `commerceSubmitOrderWithOffer`, I wonder if it's better to have its own input type, rather than expanding `CommerceSubmitOrderWithOfferInput`.

And actually, since we are creating an Exchange order first. Maybe the `note` and `artworkId` are already available from the created order, and clients don't need to pass them explicitly as arguments?

Reply:
Thanks for taking a look @starsirius 🙌 

1 - I was asking myself the same question when extending, tried to keep it like this if we change something on exchange we would be automatically extending it here, but I guess I need to spread the other exchange props and use them in the args to make it work as I expect, but even though you think that we need to create a different one?

2 - good point, we can extend the WrapQuery to grab those inputs and use them as well, going to do it tomorrow morning!

Reply:
So we don't need to extend any input if those fields are already in the Exchange response 🙏

Reply:
@starsirius all the requested changes in [91a98eb](https://github.com/artsy/metaphysics/pull/3723/commits/91a98ebbe8651fd6269a4cd840bd88851398b95f)